docs/concepts/storage/persistent-volumes.md

```diff
@@ -517,6 +518,53 @@ parameters:
 
 * `diskformat`: `thin`, `zeroedthick` and `eagerzeroedthick`. Default: `"thin"`.
 
+2. Create a persistent volume with a disk format on a user specified datastore.
+```yaml
+kind: StorageClass
+apiVersion: storage.k8s.io/v1beta1
+metadata:
+  name: fast
+provisioner: kubernetes.io/vsphere-volume
+parameters:
+    diskformat: zeroedthick
+    datastore: VSANDatastore
+```
+
+* `diskformat`: `thin`, `zeroedthick` and `eagerzeroedthick`. Default: `"thin"`.
```

Comment:
If these are bulleted items under the numbered item, add an additional indent before each one.